imports/ui/components/Cells.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { fullName } from '/imports/api/utils';
+
+
+export const ErrorPropType = PropTypes.shape({
+  message: PropTypes.string,
+  statusCode: PropTypes.number,
+});
+
+export const Cell = ({ text, href, error }) => {
+  if (error) {
+    return (
+      <span>
+        {text}
+        {' '}
+        <span className="label label-danger" title={error.message}>
+          {error.statusCode}
+        </span>
+      </span>
+    );
+  }
+
+  if (href) {
```

Comment:
I would prefer explicit Cell type setting. And then we can create an object containing different types to match.